home.html

```diff
@@ -1,76 +1,322 @@
+            <h1>Uber Navigation</h1>
+            <ul class="company">
+              <li class="item item1">
+                <h3>Uber</h3>
+              </li>
+              <li class="item item2">
+                <h3>Lead developer</h3>
+              </li>
+              <li class="item item2">
+                <h3>2018</h3>
+              </li>
+            </ul>
+
+            <ul class="tools">
+              <li class="item item1">HTML</li>
+              <li class="item item2">CSS</li>
+              <li class="item item2">Javascript</li>
+            </ul>
+
+            <p>
+              A smart assistant to make driving more safe, efficient, and fun
+              by unlocking your most expensive computer: your car.
+            </p>
+            <div>
+              <a href="#" class="btn btn-resume mt-4">See the project</a>
+            </div>
+          </div>
+          <div class="project">
+            <img src="assets/project4.png" alt="" />
+          </div>
+        </div>
+      </section>
+
+      <section class="innerworkcontainer mt-4">
+        <div class="aboutme">
+          <section class="description">
+            <div class="me-text">
+              <h1>About my self</h1>
+              <p>
+                I’m a software developer! I can help you build a product ,
+                feature or website Look through some of my work and
+                experience! If you like what you see and have a project you
+                need coded, don’t hestiate to contact me.
+              </p>
+              <h5>Let's connect</h5>
+
+              <div class="linkcontainer">
+                <ul class="links">
+                  <li>
+                    <a href="#"><img src="assets/twitter-brands.svg" alt="" class="icons" /></a>
+                  </li>
+                  <li>
+                    <a href="#"><img src="assets/linkedin-brands.svg" alt="" class="icons" />
+                    </a>
+                  </li>
+                  <li>
+                    <a href="#"><img src="assets/medium-brands.svg" alt="" class="icons" /></a>
+                  </li>
+                  <li>
+                    <a href="#"><img src="assets/github-brands.svg" alt="" class="icons" /></a>
+                  </li>
+                  <li>
+                    <a href="#"><img src="assets/angellist-brands.svg" alt="" class="icons" /></a>
+                  </li>
+                </ul>
+              </div>
+
+             <div class="btn-resume">
+              <a href="#" class="btn ">Get my resume</a>
+             </div>
+            </div>
+          </section>
+          <div class="tools-skills">
+            <ul class="list">
+              <li class="listitem expanded">
+                <div class="txtcontainer">
+                  <h3>Languages</h3>
+                </div>
+                <div class="imgcontainer">
+                  <img src="assets/expand.svg" alt="" width="20" height="20" />
+                </div>
```

Comment:
- [ ] [OPTIONAL] It was good to add the chevron icon near the languages, however, the chevron icon seems not to be well aligned with the `<h3>` languages as in the original design: 

- Your design
![image](https://user-images.githubusercontent.com/55465133/129399886-2606689e-6afe-43e6-9fec-6bb14f09291e.png)

- The original
![image](https://user-images.githubusercontent.com/55465133/129399957-e2a2d94e-0d14-4da6-a14e-4483bdd4c8ed.png)

Please let's fix this issue 😉